src/aleph/sdk/conf.py

```diff
@@ -33,6 +33,11 @@ class Settings(BaseSettings):
 
     CODE_USES_SQUASHFS: bool = which("mksquashfs") is not None  # True if command exists
 
+    CACHE_DB_PATH: Path = Field(
```

Comment:
We can be very explicit here.
```suggestion
    CACHE_DATABASE_PATH: Path = Field(
```